client/latis.py

```diff
@@ -4,6 +4,22 @@
 import urllib.parse
 
 
+def readNumpyLatis(baseUrl, latis3, dataset,
+                   projection=[], selection=[], operation=[]):
+    instance = LatisInstance(baseUrl, latis3)
+    dsObj = instance.getDataset(dataset)
+    dsObj.buildQuery(projection, selection, operation)
```

Comment:
We'll chat about this as a group but to get my thought out early: I'd expect `buildQuery()` to return a String query URL, rather than build up a stateful collection of operations to be applied to the dataset. I'd recommend separate `Dataset` functions for the different operations (e.g. `dataset.project(vars)`, `dataset.select(startTime, endTime)`, `dataset.operations(ops)`,  etc.) to build up that state, then let `buildQuery()` return a fully-formed query URL for users who would rather work with a URL than our Python objects.

I'm also trying to think about naming. I like how simple [HAPI's main read function](https://github.com/hapi-server/client-python#basic-example) is: 
```
data, meta = hapi(server, dataset, parameters, start, stop, **opts)
```

To reduce this to a single function, I might want to bring in `suffix` here as a concept and let pandas/numpy be available suffixes ("csv", "json", "pandas", "numpy", etc.). Not totally sure how to make that play nicely with file downloads vs Python object creation though.

Reply:
I split build query into `project`, `select` and `operate`. Build query still returns a string.
Also condensed the shortcut functions into `data`.  `returnType` is used to handle switching between currently only Numpy and Pandas.

`latis.data(....)` seems pretty decent for naming conventions but we can chat as a group about that.

Maybe also returning metadata similar to HAPI would be a good thing to add?